app/src/features/admin/users/ManageUsersPage.tsx

```diff
@@ -43,7 +43,8 @@ const ManageUsersPage: React.FC = () => {
     const getAccessRequests = async () => {
       const accessResponse = await restorationTrackerApi.admin.getAccessRequests([
         AdministrativeActivityStatusType.PENDING,
-        AdministrativeActivityStatusType.REJECTED
+        AdministrativeActivityStatusType.REJECTED,
+        AdministrativeActivityStatusType.ACTIONED
```

Comment:
Do we want to show actioned here?

Anyone that is approved would be in the users list below.